test/quantization/fx/test_quantize_fx.py

```diff
@@ -2423,6 +2423,120 @@ def forward(self, x):
             mp(torch.rand(4, 4, 4, 4))
             mc = convert_fx(mp)
 
+    def _check_not_observed(self, model: nn.Module, node_name_list: List[str]):
+        # check that no obsersvers have the named nodes as an input
+        for node in model.graph.nodes:
+            if hasattr(model, node.name) and isinstance(
```

Comment:
can we just check the number of observers instead? and also we can make sure the model runs

Reply:
Can you clarify these comments? The important part is whether any of the named nodes got observed, checking the number of observers doesn't really check the same thing. Its definitely a good idea to see if the model runs though, I'll add that.

Reply:
`node.name` is not a reliable thing, fx is free to change it, so the test may fail when fx tries to change it, since we are hardcoding the node name here